helpers/__tests__/convert.test.ts

```diff
@@ -0,0 +1,53 @@
+import { base64UrlToArrayBuffer, arrayBufferToBase64Url } from "../convert";
+
+function testCodecRoundTrip<T extends BufferSource, E = string>(
+  encoder: (arg: T) => E,
+  decoder: (arg: E) => T
+) {
+  for (let i = 0; i < 100; i++) {
+    // Generate a random [50, 100] buffer.
+    let buffer = new Uint8Array(Math.floor(Math.random() * 50 + 50));
+    for (let j = 0; j < buffer.byteLength; j++) {
+      buffer[j] = Math.random() * 256;
+    }
+
+    let encoded: E = encoder(buffer.buffer as T);
+    let decoded: T = decoder(encoded);
+
+    let decodedBuffer = ArrayBuffer.isView(decoded)
+      ? decoded
+      : Buffer.from(decoded);
```

Comment:
**NOTE**: Actual logical change here.

The test was failing with the following error:
```
    helpers/__tests__/convert.test.ts:17:37 - error TS2769: No overload matches this call.
      The last overload gave the following error.
        Argument of type 'BufferSource' is not assignable to parameter of type 'WithImplicitCoercion<string> | { [Symbol.toPrimitive](hint: "string"): string; }'.
          Type 'ArrayBufferView' is not assignable to type 'WithImplicitCoercion<string> | { [Symbol.toPrimitive](hint: "string"): string; }'.

    17     let decodedBuffer = Buffer.from(decoded);
```

Which makes me wonder if the test was actually running when we added it to Alberto.

Reply:
I made sure the test was failing before fixing it. So yes, it was running. I'm wondering if you have the right versions of dependencies.